config/crd/bases/wildfly.org_wildflyservers.yaml

```diff
@@ -209,6 +209,73 @@ spec:
                 minItems: 1
                 type: array
                 x-kubernetes-list-type: atomic
+              livenessProbe:
+                description: LivenessProbe defines the periodic probe of container
+                  liveness. Container will be restarted if the probe fails
```

Comment:
minor, all other descriptions not ending with url have trailing `.`

Reply:
I've added the missing trailing `.`

Reply:
But I forgot to generate the CDR :(, I'll do it now